.gitignore

```diff
@@ -6,6 +6,8 @@ dist
 npm-debug.log
 
 bundle*.js
+model/version.go
```

Comment:
I don't like ignoring the version.go file. An idea might be to use the -ldflags of the go build command to pass the -X option to the linker. This allows assignment of variables. https://golang.org/cmd/link/